client/dashboard/utils/site-features.ts

```diff
@@ -13,6 +13,13 @@ export function hasAtomicFeature( site: Site, feature: `${ DotcomFeatures }` ) {
 	return site.is_wpcom_atomic && ! site.plan?.expired && hasPlanFeature( site, feature );
 }
 
+export function hasHostingFeature( site: Site, feature: `${ DotcomFeatures }` ) {
```

Comment:
I felt a bit confused when to use this function. Would it be better to have a list that just needs to check whether the feature is in `site.plan.features.active` and handle it internally?

Reply:
> We can always use hasHostingFeature() in place of hasAtomicFeature() everywhere though, and not export hasAtomicFeature(). I think that's cleaner

I'll do it in a separate PR as it will touch many unrelated files 😄